firmware/application/ui_navigation.cpp

```diff
@@ -689,9 +689,21 @@ ReceiversMenuView::ReceiversMenuView(NavigationView& nav) {
 
     add_apps(nav, *this, RX);
 
-    for (auto const& gridItem : ExternalItemsMenuLoader::load_external_items(app_location_t::RX, nav)) {
-        add_item(gridItem);
-    };
+    auto externalItems = ExternalItemsMenuLoader::load_external_items(app_location_t::UTILITIES, nav);
```

Comment:
You could create a helper and not duplicate this code two times ;)

Reply:
Thx! will fix

Reply:
Helper added. I don’t really know how it works tho.